src/components/alert/alert.twig

```diff
@@ -1,7 +1,19 @@
 {% set tag_name = tag_name|default('div') %}
 
-<{{ tag_name }} class="c-alert{% if full_width %} c-alert--full-width{% endif %}{% if class %} {{ class }}{% endif %}">
+<{{ tag_name }} class="c-alert{% if full_width %} c-alert--full-width{% endif %} {% if floating %} c-alert__floating {% endif %}{% if class %} {{ class }}{% endif %}">
```

Comment:
Nit: Extra space and wrong separator for modifier (see earlier comments on class name)

```suggestion
<{{ tag_name }} class="c-alert{% if full_width %} c-alert--full-width{% endif %}{% if floating %} c-alert--floating {% endif %}{% if class %} {{ class }}{% endif %}">
```